c++/src/kj/async-io-unix.c++

```diff
@@ -804,6 +804,8 @@ private:
   }
 };
 
+constexpr size_t kj::AsyncStreamFd::MAX_SPLICE_LEN;
```

Comment:
You need `#if __linux__` around this to match the declaration.

Reply:
ooooh that explains things :facepalm: